docs/v2/README.md

```diff
@@ -75,6 +75,99 @@ This task will create a `Docker` image. To run a container with the `HMDA Platfo
 
 The same approach can be followed to build and run Docker containers for the other microservices that form the HMDA Platform
 
+### CI/CD in Kubernetes (local)
+
+To build and run the application in Kubernetes (local development), the following steps must be taken:
+
+1. Make sure that [kubectl](https://kubernetes.io/docs/tasks/tools/install-kubectl/) is installed and configured for your system
+2. Make sure that [minikube](https://kubernetes.io/docs/tasks/tools/install-minikube/) is installed and configured. When properly
+installed, you should be able to do `minikube dashboard` to open up the `kubernetes` cluster dashboard in your browser. Make sure that
+`kubectl` is properly configured to point to `minikube` when working in local development mode.
+3. Make sure that [Helm](https://helm.sh/) is installed, as well as Tiller, the server side component.
+4. Install the `Jenkins` Helm Chart, as follows:
+
+* Create namespace for `Jenkins`: 
+
+```bash
+kubectl apply -f kubernetes/jenkins-namespace.yaml
+```
+
+* Bind `default` service account to cluster admin role: 
+
+```bash
+kubectl apply -f kubernetes/cluster-admin-rolebinding.yaml
+```
+
+* First, make sure the `Helm` repo is up to date:
+
+```shell
+helm repo update
+```
+
+List Helm Charts installed:
+
+```shell
+helm list
+```
+
+In some cases, this command will fail with a permissions error. In that case, run the following:
+
+```shell
+kubectl create clusterrolebinding add-on-cluster-admin --clusterrole=cluster-admin --serviceaccount=kube-system:default
+```
+
+If the `helm list` command doesn't work (usually with an error of "connection refused"), do the following:
+
+```shell
+kubectl --namespace=kube-system edit deployment/tiller-deploy
+```
+
+And change the `automountServiceAccountToken` to `true`. Save and exit
+
+* Create Persistent Volume for Jenkins
+
+
+* Install Jenkins Chart
+
+```shell
+helm install --name jenkins -f kubernetes/jenkins-values.yaml stable/jenkins --namespace jenkins-system
+```
+
+You can access `Jenkins` by issuing `minikube service jenkins` and logging in with `admin/admin`.
```

Comment:
I had to use `minikube service -n jenkins-system jenkins` (specifying the namespace) here, otherwise I got a "services not found" error.

Reply:
You are correct, thanks. The initial version of this was deploying `Jenkins` on the `default` namespace. Will correct this shortly